### ISSUE TITLE
Insufficient contrast ratio with links

### ISSUE THREAD

**Issue by kulbhushanchand** [kulbhushanchand/kulbhushanchand.github.io-v3#11]
From the lighthouse accessibility audit, the contrast ratio can be improved. 

**Tip**: use chrome dev console to test the different color that gives sufficient contrast ratio, then change the `sass` variable for the link highlight color

![image](https://user-images.githubusercontent.com/17742733/124258133-b1ae8700-db4a-11eb-89d3-119c9caf32e7.png)
